app/core/settings.py

```diff
@@ -34,3 +34,4 @@
 GCP_ENVIRONMENT = env.str("GCP_ENVIRONMENT", "dev")
 DEAD_LETTER_TOPIC = env.str("DEAD_LETTER_TOPIC", "transformer-dead-letter-dev")
 MAX_EVENT_AGE_SECONDS = env.int("MAX_EVENT_AGE_SECONDS", 86400)  # 24hrs
+TRACING_ENABLED = env.str("TRACING_ENABLED", True)
```

Comment:
`TRACING_ENABLED`  is already defined in settings, few line above:
https://github.com/PADAS/cdip-routing/blob/6db0bb5d8825689d6e688fb8d5f40aa52b0a1afa/app/core/settings.py#L30C1-L30C52